Gemfile

```diff
@@ -1,2 +1,3 @@
 source "https://rubygems.org"
 gemspec
+gem "rails-controller-testing"
```

Comment:
> Re: The rails-controller-testing gem: Why do we need it? We haven't needed it before. ...

Oops, I was wrong about this, we already are using the `rails-controller-testing` gem. Please see our `Appraisals` file. No changes to our `Gemfile` are necessary.